src/_about/newcomer-guide.md

```diff
@@ -0,0 +1,379 @@
+
+#### Community Managers
+
+Community Managers oversee the daily operations of Quilt's official community spaces. As well as supporting the rest
+of the Community Team and acting as Moderators, they handle administrative tasks in the community spaces, lead
+interviews for Community Team applications, maintain and keep track of the tools used to keep the community spaces
+running smoothly, and create and update Community Team policies.
+
+Additionally, Community Managers lead investigations into wider social trends and controversies in the Minecraft
+modding sphere, collating information and using it to supplement the Moderation Team's decisions, often banning
+problem users before they cause any issues (or, in some cases, before they even enter our community spaces).
+
+Community Managers have responsibilities that the Moderators don't, but they aren't superior to the rest of the
+Community Team -- they have an equal say in decisions and votes, and only end up in a higher hierarchical position
+because most platforms require that to be the case.
+
+#### Moderators and Trainee Moderators
+
+Moderators are in charge of watching over Quilt's official community spaces, resolving conflicts and removing bad
+actors. They're the most important line of defense for our community spaces, enforcing Quilt's
+[rules](/community/rules/) and [Code of Conduct](/community/code-of-conduct/), and keeping our users safe and 
+discussions productive.
+
+Trainee Moderators are newly-appointed Moderators, who have recently passed the Community Team's interview and voting
+processes. They're designated separately in order to allow them time to get used to how Quilt's community spaces work,
+and generally remain in the Trainee position for around two weeks -- before their performance on the team is evaluated
+and the rest of the Community Team decides whether to promote them to a full Moderator position.
+
+#### Events Team
+
+Quilt's Events Team is in charge of planning, organising and running events in Quilt's community spaces, and sometimes
+acting as extra Moderators during busy events. They work with the rest of Quilt's teams (and the wider Minecraft
+modding community) to collate ideas and information relating to possible community events, and ensure that they come
+to fruition and go smoothly.
+
+Events Team members are not Moderators, and don't engage as Moderators outside the context of the events they're 
+running (unless that specific member of the Events Team also happens to be a Moderator or Community manager).
+
+#### Outreach Team
+
+The Outreach Team is responsible for running Quilt's social media accounts, creating and posting mixed-media relating
+to Quilt (including the [developer meetings podcast](https://anchor.fm/quiltmc-dev-meetings/)), and producing blog posts
+and articles for this website. They're Quilt's connection with the rest of the world in many cases, and the work they
+do is incredibly important.
```

Comment:
> and the work they do is incredibly important.

Including this here and not in the other teams' descriptions makes outreach sound more important than the other teams